woocommerce/woocommerce-ios
# Fix: Overly Large Instruction Image on application password tutorial on iPad

Closes: #12732

## Description
This PR resolves the issue reported in #12732, where the instructional image on application password tutorial on the iPad login page was excessively large, leading to a suboptimal user experience.

### Changes
- Set the maximum width of the instructional image to 400 px to ensure it fits within the designated area on the iPad screen.
- Aligned the image to the left side of the login view to maintain consistency with the app's design language.

### Testing Steps
1. Cancel the app, so you can see the tutorial.
2. Try to login into your store with the application password credentials, and you will see the application password tutorial.
3. Check that the image is aligned on the left and it has the right size both on iPad and iPhone.


## Screenshots
|  After on iPad | After on iPhone | 
:-------------------------:|:-------------------------:
![Simulator Screenshot - iPad Pro (11-inch) (4th generation) - 2024-05-16 at 18 40 44](https://github.com/woocommerce/woocommerce-ios/assets/495617/56875a88-77d0-492f-86df-0f77b20306b0) | ![Simulator Screenshot - iPhone 15 Pro Max - 2024-05-16 at 19 02 28](https://github.com/woocommerce/woocommerce-ios/assets/495617/cd249f9e-3a26-4fe5-9412-bfe9947d81ca)




---
